src/components/Organisms/TodoCard/index.jsx

```diff
@@ -26,10 +26,24 @@ export const TodoCard = () => {
     } else {
       const mappedOnTaskNameChange = [...taskList];
       mappedOnTaskNameChange[index].name = value;
+      mappedOnTaskNameChange[index].initializing = false;
       setTaskList(mappedOnTaskNameChange);
     }
   };
 
+  useEffect(() => {
+    const takeData = localStorage.getItem("taskList");
+    if (takeData !== null) {
+      const takeTaskList = JSON.parse(takeData);
+      setTaskList(takeTaskList);
+    }
+  }, []);
+
+  useEffect(() => {
+    const storeTask = JSON.stringify(taskList);
+    localStorage.setItem("taskList", storeTask);
```

Comment:
```suggestion
    const taskListJSON = JSON.stringify(taskList);
    localStorage.setItem("taskList", taskListJSON);
```
こちらもJSON化されたtaskListということで、こんな感じの変数名でも良いかと思います！

もしくは、この程度であれば一時変数に入れないほうが返って可読性が上がるかもですね
```suggestion
    localStorage.setItem("taskList", JSON.stringify(taskList));
```